index.html

```diff
@@ -0,0 +1,150 @@
+        </h1>
+        <p class="intro-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+            Impedit aspernatur deleniti assumenda dignissimos nulla fuga,
+            deserunt magnam? Tenetur vitae nisi optio veritatis numquam neque velit quasi.
+            Doloribus sed ratione nobis.</p>
+            <h3 id="date">2022.10.26(WED) ~ 28(FRI)</h3>
+            <p id="date-location">@ICC Central Business District, Abuja</p>
+    </section>
+
+    <section class="program-info" id="program">
+        <h2 id="main-program">Main Program</h1>
+        <div class="program-bar"></div>
+        <div class="program">
+        <article class="program-list">
+            <img class="program-icon" src="img/program_icon_01.png" alt="drum sticks icon">
+            <h4 class="program-topic">Drum Chops Presentation</h4>
+            <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.sunt saepe veniam sequi eum.</p>
+        </article>
+        <article class="program-list">
+            <div class="program-icon-wrapper">
+                <img class="program-icon" src="img/music-media.png" alt="drum sticks icon">
+            </div>
+            <h4 class="program-topic">Advertising & Media</h4>
+            <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Molestiae qui odit fugiat nemo maiores earum, sunt saepe veniam sequi eum.</p>
+        </article>
+        <article class="program-list">
+            <div class="program-icon-wrapper">
+                <img class="program-icon" src="img/program_icon_03.png" alt="drum sticks icon">
+            </div>
+            <h4 class="program-topic">Building Connection</h4>
+            <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Molestiae qui odit fugiat nemo maiores earum, sunt saepe veniam sequi eum.</p>
+        </article>
+        <article class="program-list">
+            <div class="program-icon-wrapper">
+                <img class="program-icon" src="img/program_icon_02.png" alt="drum sticks icon">
+            </div>
+            <h4 class="program-topic">Drum Brands Presentaion</h4>
+            <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Molestiae qui odit fugiat nemo maiores earum, sunt saepe veniam sequi eum.</p>
+        </article>
+        <article class="program-list">
+            <div class="program-icon-wrapper">
+                <img class="program-icon" src="img/program_icon_01.png" alt="drum sticks icon">
+            </div>
+            <h4 class="program-topic">Drum Brands</h4>
+            <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Molestiae qui odit fugiat nemo maiores earum, sunt saepe veniam sequi eum.</p>
+        </article>
+        </div>
+        <a class="see-more" href="#">SEE THE WHOLE PROGRAMS</a>
+    </section>
```

Comment:
**OPTIONAL** - This part of your project doesn't seem to match what is in the design template. Please try to fix this 🙏 🙏 .

Your project:
<img width="570" alt="right aligned" src="https://user-images.githubusercontent.com/73607512/196935122-a1de31c3-77c1-4499-a6f8-62c171a5efb8.PNG">

Design Template:
<img width="501" alt="center" src="https://user-images.githubusercontent.com/73607512/196935256-3be3ac6a-5057-4c27-886c-7fda2585d94f.PNG">